packages/client/src/elements/inputs/Select/Select.styled.ts

```diff
@@ -19,21 +20,21 @@ export const SelectInput = styled(IconButton)`
   }
 `;
 
-export const SelectList = styled.div<{ isOpen: boolean }>`
+export const SelectList = styled.div<{ isOpen: boolean; openDirection: SelectOpenDirection }>`
   position: absolute;
-  top: -15px;
+  top: ${({ openDirection }) => (openDirection === 'down' ? '-15px' : '-180px')};
   max-height: 317px;
   min-width: 50px;
   background: ${({ theme }) => theme.palette.background.paper};
   box-shadow: ${({ theme }) => theme.palette.boxShadow.getBoxShadow(0.15)};
   border-radius: ${({ theme }) => theme.sizes.borderRadius};
   overflow: hidden;
   transition: transform 150ms ease-in-out;
-  transform-origin: top;
-  ${({ isOpen }) =>
+  transform-origin: ${({ openDirection }) => (openDirection === 'down' ? 'top' : 'bottom')};
+  ${({ isOpen, openDirection }) =>
     isOpen
       ? `
-      transform: scaleY(1);
+      transform: ${openDirection === 'down' ? 'scaleY(1)' : 'scaleY(1)'};
```

Comment:
its doesnt make sense for me. It will always be the same `scaleY(1)` value. Please check this.

Reply:
yeap, it's a residue after testing